src/utils/moment_constants.js

```diff
@@ -28,10 +28,10 @@ export function getAllMonthsWithinRange(endpoints, index, selectedDateRange) {
     monthResults.push(referenceDate.format('YYYY-MM'));
   }
   monthResults.forEach(monthValue => {
-    if (index === endpoints.result_index) {
-      queryString += `${endpoints.prefix + index + monthValue}-*,`;
+    if (index === endpoints.indices.result_index) {
+      queryString += `${index + monthValue}-*,`;
     } else {
-      queryString += `${endpoints.prefix + index + monthValue},`;
+      queryString += `${index + monthValue},`;
     }
```

Comment:
This would be DRYer:
```
    queryString += `${index + monthValue}`;
    if (index === endpoints.indices.result_index) {
      queryString += `-*`;
    }
    queryString += `,`;
```

Reply:
Yes, maybe, although building the string through three separate steps isn't as readable. This code will go away once we can drop the remaining direct Elasticsearch queries, anyway.